news/2017-10-11-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: October 2017"
+date: 2017-10-11 09:00:00 +0000
+---
+
+There’s been major developments in the Quality Assurance Team and Beatmap Nominators this month, with new QAT leaders, changes to the Nominator Tier system, revival of mapping contests, and more ranked maps per day! Curious to know more? Read on to find out!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+Hello again everyone! Another month has passed, so here we are with the second issue of the **QAT Gazette**!
+This month was full of events going on in the Quality Assurance Team, the Beatmap Nominators, and of course osu! itself. New Beatmap Nominators have been introduced for osu!taiko, osu!catch and osu!mania, the QAT and Beatmap Nominator Upheaval and the return of Mapping with Rewards are just some of the few topics in this month’s issue, so let’s begin!
```

Comment:
IT'S NITPICK TIME WEEWOO

full of events going on is kind of redcurrant, events infers that things are already happening. you could probably drop this whole line and just use the New Beatmap Nominators.. line after it, reads better